crates/rpc/rpc/src/validation.rs

```diff
@@ -365,6 +366,29 @@ where
 
         Ok(versioned_hashes)
     }
+    /// Validates the given [`BlobsBundleV1`] and returns versioned hashes for blobs.
+    pub fn validate_blobs_bundl_v2(
+        &self,
+        mut blobs_bundle: BlobsBundleV2,
+    ) -> Result<Vec<B256>, ValidationApiError> {
+        if blobs_bundle.commitments.len() != blobs_bundle.proofs.len() ||
+            blobs_bundle.commitments.len() != blobs_bundle.blobs.len()
+        {
+            return Err(ValidationApiError::InvalidBlobsBundle)
+        }
```

Comment:
We need to check `len(commitments) == len(blobs)` and `len(proofs) = len(blobs) * CELLS_PER_EXT_BLOB`